sdk/core/core-client-rest/review/core-client.api.md

```diff
@@ -4,12 +4,16 @@
 
 ```ts
 
+/// <reference types="node" />
```

Comment:
can we define some compatible interfaces to avoid needing to export shapes from node's `tls` module?